Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11689,6 +11797,40 @@ end
 					},
 					Notes = "Returns true if the specified player is queued to be transferred to a World.",
 				},
+				RegisterForgeMod =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list.",
+				},
+				RegisterForgeModForProtocol =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+						{
+							Name = "ProtocolVersionNumber",
+							Type = "number",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list for all protocol versions.",
```

Comment:
Fixed API description in https://github.com/satoshinm/cuberite/pull/2/commits/d7e5e5858d391a0ca4cc2f554c1a4ce8c34124fa